source/règles/externalized.yaml

```diff
@@ -1,3 +1,18 @@
+contrat salarié . indemnité kilométrique vélo:
+  description.en: >-
+    Employer's incitation to the use of bicycles for commuting.
+    This financial aid is not taxed.
+  titre.en: kilometric bicycle allowance
+contrat salarié . indemnité kilométrique vélo . indemnité vélo active:
+  question.en: | 
+    Can the employee take advantage of the kilometric bicycle allowance to go to work ?
```

Comment:
Bizarre comme formulation, je mettrai plutôt : 
Does the employee receive the kilometric bicycle allowance ?

Reply:
Quitte à détailler les conditions dans la description

Reply:
OK j'ai mis does au lieu de can